sapbtp-operator-charts/values.yaml

```diff
@@ -38,6 +38,12 @@ manager:
     req_cpu_limit: ""
   imagePullSecrets: []
 #   - name: registrySecretName
+#  imagePullSecretName: registrySecretName
```

Comment:
Hi Gadi, I want to test this, can you please add an example of how to set these values using --set?

Reply:
Using `--set` is a bit limited for array with inner map.
How about using instead `--values` (or `-f`) with local file which has these values?
Both `helm install` and `helm template` support it.
See for example the docs of `helm template`: https://helm.sh/docs/helm/helm_template/
Example values file: (note that the auth' line should be: "- authentication:"

manager:
  imagePullSecretName: registrySecretName
  pullSecrets: []
  - authentication:
      password: mypwd
     username: myuser
    host: myhost

Reply:
please review this PR
https://github.com/SAP/sap-btp-service-operator/pull/252/files

Reply:
PR 252 looks good, thanks

Reply:
thanks, I merged it and it will be released until EOW